### PR TITLE
Move "Tips & More Info" link from homepage to top menu bar

### PR DESCRIPTION
**Pull request by Iniquit**
Hi! It's exciting to see IFDB go open source! This PR addresses [ifdb-suggestion-tracker issue #116](https://github.com/iftechfoundation/ifdb-suggestion-tracker/issues/116). Also standardizes spaces between "curarrow" icon and the menu items next to it (made them all use a single space, like 'Home' uses now.)

**Before:**
before

**After:**
after

### CODE REVIEW

File: www/home [hunk 1]
```diff
@@ -65,7 +65,6 @@ pageHeader("The Interactive Fiction Database - IF and Text Adventures");
          recommendation engine.  IFDB is a Wiki-style
          community project: members can add new game listings, write
          reviews, exchange game recommendations, and more.
-         <span class=details><a href="tips">Tips &amp; More Info</a></span>
```
Comment (dfabulich) [inline www/home:68]: I wonder if we should add this back? It's OK to have multiple links, especially if our goal is to make that page more visible.

**Comment by dfabulich** [comment]
This commit is currently in conflict with the main branch _and_ with your responsive home page branch! 😬  We may want to let the responsive home page branch go through first, and then reapply this PR again later when the header is refactored and the code is settled down…?

File: www/home (hunk 1 above)
Comment (Iniquit) [reply www/home:68]: Works for me!

**Comment by Iniquit** [comment]
Sounds like a good idea! This one is super small anyway.

**Comment by dfabulich** [comment]
I'm going to close this old conflicting PR. Feel free to reopen in the future.

**Pull request closed without merging**